Gemfile.jruby-3.1.lock.release

```diff
@@ -8,28 +8,37 @@ PATH
   remote: logstash-core
   specs:
     logstash-core (8.10.0-java)
+      cgi (~> 0.3.6)
```

Comment:
review note: default gems don't normally appear in the Gem's lockfile, but since we declare versioned dependencies in the `gemspec` they appear here in the resolved lock. When we upgrade to Jruby 9.4.x that includes them, they will be removed from the `gemspec`, which will cause them to disappear from this lock-file.